target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -47,6 +47,10 @@ comfast,cf-e120a-v3)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "$boardname:green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "$boardname:green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-wr650ac-v1|cf-wr650ac-v2)
```

Comment:
@grosjo This is not going to work for v2, anyway you can drop this and add `linux,default-trigger = "phy0tpt";` to your DTS files instead.

Reply:
You don't need to use `01_leds` at all if you add `linux,default-trigger = "phy0tpt"` and `linux,default-trigger = "phy1tpt"` triggers to your LEDs in your DTS file as you've it [done already in your other PR](https://github.com/openwrt/openwrt/pull/1891/commits/903f34c8c64f4d78ea4cb25178ce22806f868c64#diff-285ac400812ddee4d019c28256ada29dR59)

Reply:
Ok, I fixed this one